api/src/utils/media/xlsx/transformation/xlsx-transform.ts

```diff
@@ -178,6 +230,7 @@ export class XLSXTransform {
         return RowObject[columnName];
       })
       .filter(Boolean)
+      .filter(Boolean)
```

Comment:
Is the double `.filter(Boolean)` intentional?

Reply:
Nope! I think I had changes across 2 PRs, so probably just a funny merge. Will fix.

Reply:
Fixed